src/shared/components/query/quickSearch/QuickSearch.tsx

```diff
@@ -278,10 +289,10 @@ export default class QuickSearch extends React.Component {
     render() {
 
         return (
-            <div>
+            <div onFocusCapture={this.onFocus}>
                 <Select
                     options={this.options.result || []}
-                    autoFocus={true}
+                    autoFocus={false}
```

Comment:
@dippindots why does auto focus need to be disabled? I like that one :) - allows you to query immediately from the homepage without having to click on the searchbox

Reply:
I can turn it on, I am using the focus event to trigger the menu now. I turned off this because we will have an empty menu showing up when we landing to the quick-search tab. Should we allow this?

Reply:
@dippindots i see - maybe we can hide the popup menu when the search bar is empty?